src/figuresClasses.ts

```diff
@@ -1,19 +1,99 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
+const error = new Error('somethig went wrong');
```

Comment:
no, you want your errors to be maximally descriptive. I as a user comes here - passed irrelevant input. I don't know what I did wrong. But I should know